docker/docker-compose.yaml

```diff
@@ -29,6 +29,8 @@ services:
       # - CODEX_MARKETPLACE_ADDRESS=${CODEX_MARKETPLACE_ADDRESS:-0x59b670e9fA9D0A427751Af201D676719a970857b}
       - CODEX_VALIDATOR=${CODEX_VALIDATOR:-false}
       - CODEX_VALIDATOR_MAX_SLOTS=${CODEX_VALIDATOR_MAX_SLOTS:-1000}
+      - NAT_IP_AUTO=false
+      - NAT_PUBLIC_IP_AUTO=https://ipinfo.io/ip
```

Comment:
Are we sure we're going to be able to rely on this service? It seems like it would be kinda flaky. I strongly suggest we host our own or as has been suggested rely on libp2p infrastructure for this.

Reply:
We do not rely on it?

https://github.com/codex-storage/nim-codex/blob/750fe2392feb57e6043b824e8e939a99d866e30a/docker/docker-entrypoint.sh#L9-L15

We are using a generic approach and a service defined in the `NAT_PUBLIC_IP_AUTO` variable will be used.

Looks like we specified `https://ipinfo.io/ip` in [docker-compose.yaml](https://github.com/codex-storage/nim-codex/blob/master/docker/docker-compose.yaml). But it is fine for now and just as an example?

And we have a task to build our own - https://github.com/codex-storage/infra-codex/issues/91

Reply:
Ah, great - yeah, it's a bit confusing to have it default to that, not sure whats better - maybe no default. I'm sure our IPs are gonna get banned pretty quickly if we run a large number of nodes or too many frequent tests against it.

Reply:
Got you, thanks for clarifying.